src/sdk-reference/js/6/offline-tools/index.md

```diff
@@ -0,0 +1,76 @@
+---
+layout: sdk.html.hbs
+title: Offline Tools
+description: Tools to handle the loss of connection to the Kuzzle server
+order: 400
+separator: essentials
+---
+
+# Offline tools
+
+The Kuzzle SDK provides a set of properties that help your application to be resilient to the loss of network connection
+during its lifespan. 
+
+## offlineQueue
+
+A read-only `Object` containing the requests queued while the SDK is in the `offline` state.
+
+## queueMaxSize
+
+A writable `number` defining the maximun size of the `offlineQueue`.
+
+## queueTTL
+
+A writable `number` defining the time in milliseconds a queued request is kept in the `offlineQueue`.
+
+## startQueuing()
+
+Starts the requests queuing. Request will be put in the `offlineQueue` instead of being discarded, until `stopQueuing` is called.
+Works only in `offline` state, and if the `autoQueue` option is set to false. Call `playQueue` to send to Kuzzle the
+requests in the queue, once the SDK state passes to `online`. Call `flushQueue` to empty the queue without sendint the requests.
+
+## stopQueuing()
+
+Stop queuing the requests. Requests will no more be put in the `offlineQueue`, they will be discarded.
+Works only in the `offline` state, and if the `autoQueue` option is set to `false`.
+
+## playQueue()
+
+Sends to Kuzzle all the requests in the `offlineQueue`. Works only if the SDK is not in a `offline` state, and if the 
+`autoReplay` option is set to false.
+
+## flushQueue()
+
+Empties the `offlineQueue` without sending the requests to Kuzzle.
+
+## autoQueue
+
+A writable `boolean` telling the SDK whether to automatically queue requests during the `offline` state or not.
+
+## autoReplay
+
+A writable `boolean` telling the SDK whether to automatically send or not the requests in the `offlineQueue` on a
+`reconnected` event.
+
+## autoReconnect
+
+A writable `boolean` telling the SDK whether to automatically reconnect or not to Kuzzle after a connection loss.
+
+## reconnectionDelay 	
+
+A read-only `number` specifying the time in milliseconds between different reconnection attempts.
```

Comment:
Is this this configurable somehow?

Reply:
You can only set it as a constructor option.

Reply:
Couldn't we point out where and how this can be configured?

Reply:
It is explained in the constructor page

Reply:
ok, we can find it easily enough with the search :+1: